src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx

```diff
@@ -1,15 +1,53 @@
 import React from 'react';
 
 const VetTecApprovedPrograms = () => (
```

Comment:
Should this be `VetTecApplicationProcess`?

Reply:
Yes, thanks!